src/view/kc-display.js

```diff
@@ -0,0 +1,14 @@
+import { LitElement, html } from 'lit-element'
+
+export class KCDisplay extends LitElement {
+  render () {
+    return html`
+    <div>
+      <kc-webcam-selector></kc-webcam-selector>
```

Comment:
Do you not have to import these? 

I'm not saying this is wrong, just want to understand what's happening...

Reply:
These are declared at the browser level. They can be dynamically imported (you can see an example of the dynamic definition in spikes). in this case, they are defined in the html at the beginning in the same way. They become like normal tags like div. I think it is this aspect that I find so agreeable about this technic... for the moment.